sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+by the BTC the user submitted.  This BTC may only be spent to fulfill withdraws,
+or to process a wallet hand-off at the end of a reward cycle.  Either way, the
+user's BTC will be spent by Stackers before they can reclaim it via the spending
+fulfillment script.
+
+![Figure 1](./001-sip-025-figure-1.png)
+
+_Figure 1: Protocol diagram of an sBTC deposit.  The user submits a Bitcoin
+transaction, and eventually, the equivalent number of sBTC tokens materialize in
+the account of the user's choosing_.
+
+### Deposit Bitcoin Transaction
+
+The deposit Bitcoin transaction contains the following data:
+
+* An `OP_RETURN` data output, which contains the account address to which sBTC
+  should be minted
+
+* A payment to the sBTC sBTC wallet maintained by the current Stackers.  This
+  payment UTXO is the hash of a script that has two spending fulfillment
+conditions:  either the Stackers spend the UTXO before the end of the next
+reward cycle, or the sender can spend the UTXO once the end of the next reward
+cycle has passed.
+
+The payment must be above a `.sbtc`-defined threshold, in satoshis.  This
+_minimum deposit_ value is used to prevent users from spamming the system with
+dust deposits.  Stackers can vote to adjust the minimum deposit.
+
+### Stacker Authorization Transaction
+
+The authorization transaction contains the following data:
+
+* The quantity of BTC sent to the sBTC wallet.
+
+* The address to which the sBTC should be minted.
+
+* A vote "yes", "no", or "abstain" to carry out the deposit, to deny the deposit,
+  or to record that no agreement could be reached (respectively).
+
+* A bitmap of which stackers voted.
+
+* A pointer to the deposit transaction's UTXO
+
+, * The Bitcoin redeem script for the user's payment.  The Stackers must verify
+  that the script contains the aforementioned spending fulfillment conditions,
+_and no others_.  The script is disclosed on the Stacks chain in order to (1)
+allow 3rd parties to authenticate the script and (2) check the Stackers' work in
+order to hold them accountable.
+
+On successful execution, the sBTC will be minted to the user-specified Stacks
+account, and the redeem script will be stored in the `.sbtc` contract for
+posterity.  The `.sbtc` contract does not attempt to parse or validate the
+redeem script, but it does keep track of all of the UTXOs that the wallet
+has assumed control over via "yes" votes.  In addition, it stores the voting
+results and bitmaps for posterity.
+
+There is no transaction fee for the Stacker authorization transaction.  Stackers
+instead require the miner to include it for free in their blocks by withholding
+block signatures until it is included.
+
+### Failure Modes and Recovery
+
+There are several places where a deposit can fail.  First, the Stackers may fail
+to complete a signing round that achieves the 70% threshold.  One of the
+following outcomes will happen:
+
+* The Stackers produce a deposit authorization with at least 70% of the STX
```

Comment:
should this read 30% or more of the STX voting against.. ?

Reply:
One implies the other

Reply:
Resolved.